test/e2e/dynamic_resource_e2e_test.go

```diff
@@ -1,105 +1,163 @@
 package e2e
 
 import (
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/dynamic"
 	"strings"
 
-	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry"
 	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
 )
 
-var _ = Describe("Dynamic Resource", func() {
-	It("resolve prometheus API", func() {
+var _ = FDescribe("Dynamic Resource", func() {
```

Comment:
This will need to be unfocused before it can be merged. Ginkgo's runner will set a nonzero exit status when any of the test nodes are focused, as a safeguard against accidentally merging them, so the E2E check should be failing.